hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -45,6 +45,11 @@ grep -A1 'Crashed tests' "${REPORT_DIR}/output.log" \
   | cut -f2- -d' ' \
   | sort -u >> "${REPORT_DIR}/summary.txt"
 
+## Check if Maven was killed
+if grep -q 'Killed.* mvn .* test ' "${REPORT_DIR}/output.log"; then
```

Comment:
So we are presuming that Killed will never be used by a test? :) I am fine with that.
+1. LGTM.